zeppelin/zeppelin.sh

```diff
@@ -66,8 +72,8 @@ function install_zeppelin() {
   local asm_3_1=/usr/lib/zeppelin/lib/asm-3.1.jar
   local asm_5_0_4=/usr/lib/zeppelin/lib/asm-5.0.4.jar
 
-  if [[ "$zeppelin_version" == "0.7."* && -f "$asm_3_1" && -f "$asm_5_0_4" ]]; then
-    rm "$asm_5_0_4"
+  if [[ "${zeppelin_version}" == "0.7."* && -f "${asm_3_1}" && -f "${asm_5_0_4}" ]]; then
```

Comment:
Add a comment to explain why it is needed.

Reply:
I don't think we know why it is needed.  We just replaced $... with ${...} as you requested above

Reply:
@erickMunz - do you have context here?  I don't know Zeppelin well enough to say anything useful about this block of code.

Reply:
@cjac  not something I have worked on, but I believe this involves some version compatibility change to use libraries, but we did not changed that

Reply:
I think we should leave it without more than the superficial variable syntax changes, @functicons